src/Application/Config.php

```diff
@@ -13,15 +13,17 @@ class Config {
 	public function __construct(
 		public readonly ?string $sitelinkSiteId = null,
 		private readonly ?PropertyId $itemTypeProperty = null,
-		private readonly ?FacetConfigList $facets = null
+		private readonly ?FacetConfigList $facets = null,
+		private readonly ?array $icons = null
 	) {
 	}
 
 	public function combine( Config $config ): self {
 		return new Config(
 			$config->sitelinkSiteId ?? $this->sitelinkSiteId,
 			$config->itemTypeProperty ?? $this->itemTypeProperty,
-			$config->facets ?? $this->facets
+			$config->facets ?? $this->facets,
+			$config->icons ?? $this->icons
```

Comment:
Example of a line change that could have been avoided if the tailing comma was already there. Still, this only matters minutely